src/casp/services/api/routers/cell_operations_router.py

```diff
@@ -54,15 +54,12 @@ async def nearest_neighbor_search(
 @cell_operations_router.post(path="/query-cells-by-ids", response_model=t.List[schemas.CellariumCellMetadata])
 def get_cells_by_ids(
     item: schemas.CellariumCellByIdsInput,
-    user: models.User = Depends(dependencies.authenticate_user),
+    # user: models.User = Depends(dependencies.authenticate_user),
 ):
     """
     Get cells by their ids from a single anndata file with Cellarium CAS. Input file should be validated and sanitized
     according to the model schema.
     """
     return cell_operations_service.get_cells_by_ids_for_user(
         cell_ids=item.cas_cell_ids,
-        model_name=item.model_name,
```

Comment:
Now, we don't need a model name to get the cells. Previously we did  need it because we had bigquery datasets and we had different datasets for different models. Now we have only one main cell info repository, so we don't need to add model_name to get cells.

Reply:
Got it.  And we don't need user information?

Reply:
Nope, because previously, we used user only to validate that the model is available for the user (e.g. when model was only for admin users, we checked that it was the admin user calling it), but now there is no model to validate for the user, so no user needed as well.

Reply:
oh awesome! Cool